README.md

```diff
@@ -1,5 +1,40 @@
 [![Automatic version updates](https://github.com/ZOSOpenTools/gnport/actions/workflows/bump.yml/badge.svg)](https://github.com/ZOSOpenTools/gnport/actions/workflows/bump.yml)
 
-gn
+# gnport
 
 A port of the Google Generate Ninja (gn) meta-build system to z/OS Open Tools project
+
+
+### Developer Notes
+
+Before building, use this workflow to ensure the file system is in 
+pristine state.
```

Comment:
Suggest to replace by:
If you encounter a compiler error in zoslib, use this workflow to ensure the installed zoslib is current.

since the issue who had is not a common one and may be time-consuming unnecessarily.

Reply:
Good point.  I updated the commit.  Thanks Gaby!